apps/dashboard/app/(app)/[workspaceSlug]/apis/[apiId]/api-id-navbar.tsx

```diff
@@ -226,7 +226,8 @@ const NavbarContent = ({
 export const ApisNavbar = ({ apiId, keyspaceId, keyId, activePage }: ApisNavbarProps) => {
   const workspace = useWorkspaceNavigation();
 
-  const isMobile = useIsMobile();
+  // Default to false (desktop) to prevent hydration mismatches
+  const isMobile = useIsMobile({ defaultValue: false });
```

Comment:
This `useIsMobile` is being imported from @/hooks in the dashboard, and NOT the internal/ui `useIsMobile` that you added in this PR. 

the `useIsMobile` in internal/ui/src/hooks isn't being exported in internal/ui/src/index.ts to be imported and used either.